src/components/rich-text-editor/rich-text-editor.ts

```diff
@@ -54,7 +54,10 @@ export class MRichTextEditor extends ModulVue implements InputManagementData, In
             placeholderText: this.as<InputManagement>()!.placeholder
         };
         propOptions.toolbarStickyOffset = this.toolbarStickyOffset;
-        propOptions.scrollableContainer = this.scrollableContainer;
+
+        if (this.scrollableContainer && document.querySelector(this.scrollableContainer)) {
+            propOptions.scrollableContainer = this.scrollableContainer;
```

Comment:
A votre avis, je rajoute un warning dans la console si on trouve pas le scrollable-container?

Reply:
imaginons que tu set la prop scrollable-container=".uneClasse" et qu'aucun élément n'a la classe uneClasse. Le code froala pète, l’éditeur est pas initialisé et on a des erreurs du genre Cannot read property 'ABC' of undefined